src/assets/style.css

```diff
@@ -44,7 +44,7 @@
 
 .playkit-subtitles {
   position: absolute;
-  top: 0;
+  top: 60px;
```

Comment:
@yairans what about captions that want to be positioned in upper part of div? now they will always have -60px offset from top

Reply:
right. i don't have a better solution

Reply:
what happens in native implementations? what does safari do?

Reply:
we have a problem in native impl....

Reply:
Can u elaborate? Also, what happens with native player of safari when stream has captions in upper part of video?